gui/src/app/pyodide/AnalysisPyFileEditor.tsx

```diff
@@ -168,10 +183,27 @@ const AnalysisPyFileEditor: FunctionComponent<Props> = ({
       onSetEditedText={setEditedFileContent}
       readOnly={readOnly}
       toolbarItems={toolbarItems}
+      contentOnEmpty={contentOnEmpty}
     />
   );
 };
 
+const analysisPyTemplate = `import matplotlib.pyplot as plt
```

Comment:
I think it is worth having `print(draws)` in this template as that also gives more help text.

Reply:
so that would need to be implemented on the Python class?

Reply:
It is: https://github.com/flatironinstitute/stan-playground/blob/main/gui/src/app/pyodide/pyodideWorker/sp_load_draws.py#L37-L43

Reply:
Done